client/src/pages/Map/Map.js

```diff
@@ -85,6 +86,17 @@ export default function Map({
   const queryClient = useQueryClient();
   const selectionEnabledRef = useRef(selectionEnabled);
   const currentFeature = useRef(null);
+  const hasInitialFlyToFired = useRef(false);
+
+  const initialLoadTreeId = useRef(
```

Comment:
It's pretty key that this is in a `useRef`. This value should NOT change on any rerender since we don't want this id to respond to updates to the URL params